charts/nginx-ingress/values.yaml

```diff
@@ -64,6 +64,9 @@ controller:
   ## Enables debugging for NGINX. Uses the nginx-debug binary. Requires error-log-level: debug in the ConfigMap via `controller.config.entries`.
   nginxDebug: false
 
+  ## Shared process namespace between containers in the Ingress Controller pod.
+  sharedProcessNamespace: false
```

Comment:
should we name it `share` instead of `shared`?

Reply:
The attribute in the pod template is indeed `shareProcessNamespace` i thought it was more logic but i don't mind changing it

Reply:
> should we name it `share` instead of `shared`?

Updated on 8961b32f8f4a1a62883ba28fa92d8c88f657bf61